utils/azurescript.sh

```diff
@@ -32,6 +34,13 @@ export APP_SUBNET_RANGE=10.9.0.0/24
 export CONTAINER_SUBNET="${VIRUS_SCANNER_NAME}SN"
 export CONTAINER_SUBNET_RANGE=10.9.1.0/24
 
+## App Environment Variables
+export NOTIFY_API_BASE_URL=
+export NOTIFY_API_KEY=
+export NOTIFY_CONFIRMATION_TEMPLATE_ID=
+export REACT_APP_GOOGLE_ANALYTICS_ID=
```

Comment:
This Google Analytics one is just needed in the docker build pipeline, not here.